yt_dlp/extractor/tenplay.py

```diff
@@ -1,70 +1,76 @@
 # coding: utf-8
 from __future__ import unicode_literals
-
+from datetime import datetime
+import base64
```

Comment:
```suggestion

from datetime import datetime
import base64

```

seperate into groups